packages/react-frontend/src/MyApp.js

```diff
@@ -110,6 +111,37 @@ function MyApp() {
     return promise;
   }
 
+  function postProfile(profile) {
+    console.log(profile)
+    const promise = fetch("http://localhost:8000/profile", {
```

Comment:
Edit this to use the user database instead of a new profiles database with the API_URL backend server?